library/src/main/scala/java/nio/TypedArrayBuffer.scala

```diff
@@ -0,0 +1,53 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js API               **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-lang.org/     **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+package java.nio
```

Comment:
I really do not like that we add user visible stuff to the `java` package. This makes it unclear in user code, that this is actually Scala.js specific. Could we move this to `scala.scalajs.typedarray`?

We can make `library/java/nio/TypedArrayBufferBridge` public, but remove the `*.class` file when we package. (we already have specific logic anyway).